CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
@Pranav-Rastogi you can refer this link for ```all-contributor``` specs
https://github.com/kentcdodds/all-contributors